flocker/node/script.py

```diff
@@ -221,6 +200,9 @@ def flocker_container_agent_main():
     This starts a Docker-based container convergence agent.
     """
     service_factory = AgentServiceFactory(
+        # TODO this needs to be something which parses Options
+        # to get a hostname, the only necessary parameter for
+        # ApplicationNodeDeployer
```

Comment:
It occurs to me, that we probably want to get the remote ports to connect to, as well as the hostnames from the configuration file, rather than from command-line arguments (and thus have a configuration file shared between the container and dataset agent).